cpu/cc2538/include/cc2538_rf.h

```diff
@@ -212,7 +214,7 @@ enum {
     lock_status      = 0x19,
     pa_pd            = 0x20,
     lna_pd           = 0x2a,
-    disabled         = 0x2b,
+    disabled,
```

Comment:
All the other enum members are numbered explicitly, why remove it here?

Reply:
The other all actually have that value, I though it might be clearer that that one was to be used for disabling, but I don't care either way.

Reply:
you could have `disabled = 0xff`